agent/engine/docker_client.go

```diff
@@ -984,16 +988,17 @@ func (dg *dockerGoClient) Version() (string, error) {
 	return info.Get("Version"), nil
 }
 
-type volumeResponse struct {
-	Volume *taskresource.VolumeResource
+// Response object wrapper for CreateVolume and InspectVolume
+type VolumeResponse struct {
```

Comment:
This should be in the `agent/engine/types.go`?

Reply:
I like keeping the types where it's used the most, but I don't feel too strongly about it. I will moe it to types.